lib/transmuxer/adts.js

```diff
@@ -146,6 +146,84 @@ shaka.transmuxer.ADTS = class {
   static getFrameDuration(samplerate) {
     return (shaka.transmuxer.ADTS.AAC_SAMPLES_PER_FRAME * 90000) / samplerate;
   }
+
+  /**
+   * @param {string} codec
+   * @param {number} channelCount
+   * @return {?Uint8Array}
+   */
+  static getSilentFrame(codec, channelCount) {
+    switch (codec) {
+      case 'mp4a.40.2':
```

Comment:
Like the stuff below, is there any reference for where these values come from for `mp4a.40.2`? Like, any reference document that you got them from or ffmpeg command that generated them?

Reply:
I got the values from https://github.com/xqq/mpegts.js/blob/master/src/remux/aac-silent.js and https://github.com/video-dev/hls.js/blob/master/src/remux/aac-helper.ts